Formula/n/newnode.rb

```diff
@@ -0,0 +1,65 @@
+class Newnode < Formula
+  desc "Web proxy that uses a distributed p2p network to circumvent censorship"
+  homepage "https://www.newnode.com/newnode-vpn"
+  url "https://github.com/clostra/newnode.git",
+    tag:      "2.1.5",
+    revision: "18a3f713267b2a08e34cbe04253df891889997a2"
```

Comment:
```suggestion
      tag:      "2.1.5",
      revision: "18a3f713267b2a08e34cbe04253df891889997a2"
```
Align the arguments of `url`. Though, we really should use a tarball instead of a Git checkout.